### PR TITLE
Remove novaXR ip from controlpanel

### PR DESCRIPTION
**Pull request by daniellasry**
nova xr ip text box is not in cp5 anymore

### CODE REVIEW

**Comment by Andrey1994** [comment]
Can we just remove this text input?

**Comment by daniellasry** [comment]
I could do that instead.

File: OpenBCI_GUI/ControlPanel.pde [hunk 1]
```diff
@@ -2201,32 +2198,15 @@ class NovaXRBox {
         x = _x;
         y = _y;
         w = _w;
-        h = 127 + _padding;
+        h = 103 + _padding;
```
Comment (retiutut) [inline OpenBCI_GUI/ControlPanel.pde:2201]: `h = 104;`

File: OpenBCI_GUI/ControlPanel.pde [hunk 2]
```diff
@@ -2201,32 +2198,15 @@ class NovaXRBox {
...
-            .setColorForeground(isSelected_color)  // border color when not selected
-            .setColorActive(isSelected_color)  // border color when selected
-            .setColorCursor(color(26, 26, 26))
-            .setText(novaXR_ipAddress)
-            .align(5, 10, 20, 40)
-            .onDoublePress(cb)
-            .setVisible(true)
-            .setAutoClear(true); 
-        novaXR250 = new Button (x + w - padding*2 - 60*2, y + 14 + padding*3 + 26, 60, 24, "250Hz", fontInfo.buttonLabel_size);
+        novaXR250 = new Button (x + w - padding*2 - 60*2, y + 16 + padding*3, 60, 24, "250Hz", fontInfo.buttonLabel_size);
```
Comment (retiutut) [inline OpenBCI_GUI/ControlPanel.pde:2206]: ` novaXR250 = new Button (x + w - padding*2 - 60*2, y + 16 + padding*2, 60, 24, "250Hz", fontInfo.buttonLabel_size);`

File: OpenBCI_GUI/ControlPanel.pde [hunk 3]
```diff
@@ -2201,32 +2198,15 @@ class NovaXRBox {
...
         novaXR250.setHelpText("Set Sampling Rate to 250Hz.");
         novaXR250.setColorNotPressed(isSelected_color);
-        novaXR500 = new Button (x + w - padding - 60, y + 14 + padding*3 + 26, 60, 24, "500Hz", fontInfo.buttonLabel_size);
+        novaXR500 = new Button (x + w - padding - 60, y + 16 + padding*3, 60, 24, "500Hz", fontInfo.buttonLabel_size);
```
Comment (retiutut) [inline OpenBCI_GUI/ControlPanel.pde:2209]: `novaXR500 = new Button (x + w - padding - 60, y + 16 + padding*2, 60, 24, "500Hz", fontInfo.buttonLabel_size);`

File: OpenBCI_GUI/ControlPanel.pde [hunk 4]
```diff
@@ -2251,8 +2231,7 @@ class NovaXRBox {
         text(boxLabel, x + padding, y + padding);
         textAlign(LEFT, TOP);
         textFont(p4, 14);
-        text(ipTextLabel, x + padding, y + padding*2 + 17);
-        text(sampleRateLabel, x + padding, y + padding*3 + 19 + 24);
+        text(sampleRateLabel, x + padding, y + padding*3 + 19);
```
Comment (retiutut) [inline OpenBCI_GUI/ControlPanel.pde:2234]: `text(sampleRateLabel, x + padding, y + padding*2 + 18);`

File: OpenBCI_GUI/ControlPanel.pde [hunk 5]
```diff
@@ -2201,32 +2198,15 @@ class NovaXRBox {
         x = _x;
         y = _y;
         w = _w;
-        h = 127 + _padding;
+        h = 104 + _padding;
```
Comment (retiutut) [inline OpenBCI_GUI/ControlPanel.pde:2201]: Just `h = 104;` , adding padding here adds 1 too many paddings, as this is already part of 104.

**Comment by retiutut** [comment]
Need this merged next to help #768

**Pull request merged**